wpilibc/src/main/native/include/frc/Errors.h

```diff
@@ -52,13 +53,33 @@ const char* GetErrorMessage(int32_t* code);
  * Generally the FRC_ReportError wrapper macro should be used instead.
  *
  * @param status error code
- * @param message error message details
  * @param fileName source file name
  * @param lineNumber source line number
  * @param funcName source function name
+ * @param format error message format
+ * @param args error message format args
+ */
+void ReportErrorV(int32_t status, const char* fileName, int lineNumber,
+                  const char* funcName, fmt::string_view format,
```

Comment:
What's the difference between fmt::string_view and std::string_view? Is fmt::string_view just polyfill or does it do something extra?